ddcz/templates/discussions/phorum-list.html

```diff
@@ -7,36 +7,59 @@
     <h1 class="page-heading">Fórum</h1>
 
-<p>Fórum slouží především pro informace masového charakteru a informace obecně 
-zajímavé, zvláště ty, co se nedají moc rozumně umístit jinam. 
-Např. pozdravy z Conů, oznamení o významných novinkách na tematicky příbuzných 
-serverech, informace o akcích komunitního charakteru (Jdeme s Pepou do kina na 
-Narnii, chcete taky?) atp. Toto fórum slouží i jako guestbook.<br />Nachází se zde 
-také rozcestník k několika nejzákladnějším stolům, které by měly Fóru ulehčit, 
-a proto je prosím používejte, stejně jako mnoho dalších v Putyce 
-(nikdo vám hlavu neutrhne, když tak neučiníte, ale tady vaše oznámení rychleji 
-zapadne).<br /> Za obsah příspěvku ručí autor, nikoli vlastník serveru nebo redakce 
-DraciDoupe.cz. Toto ovšem neznamená, že nevhodné (např. vulgární) příspěvky 
-nebudou bez diskuse mazány, byť mazat se bude jen v nejnutnějších případech.<br />
-<b>Mějte na paměti, že Fórum je viditelné pro kohokoliv &mdash; je to naše vizitka.</b></p>
+    <p class="annotation">
+        Fórum slouží především pro informace masového charakteru a informace obecně
+        zajímavé, zvláště ty, co se nedají moc rozumně umístit jinam.
+        Např. pozdravy z Conů, oznamení o významných novinkách na tematicky příbuzných
+        serverech, informace o akcích komunitního charakteru (Jdeme s Pepou do kina na
+        Narnii, chcete taky?) atp. Toto fórum slouží i jako guestbook.
+    </p>
+    <p>
+        Nachází se zde
+        také rozcestník k několika nejzákladnějším stolům, které by měly Fóru ulehčit,
+        a proto je prosím používejte, stejně jako mnoho dalších v Putyce
+        (nikdo vám hlavu neutrhne, když tak neučiníte, ale tady vaše oznámení rychleji
+        zapadne).
+    </p>
+    <p>
+        Za obsah příspěvku ručí autor, nikoli vlastník serveru nebo redakce
+        DraciDoupe.cz. Toto ovšem neznamená, že nevhodné (např. vulgární) příspěvky
+        nebudou bez diskuse mazány, byť mazat se bude jen v nejnutnějších případech.
+    </p>
+    <p>
+        <b>Mějte na paměti, že Fórum je viditelné pro kohokoliv &mdash; je to naše vizitka.</b>
+    </p>
+
+    {% if user.is_authenticated %}
+        <form method="post" action="{% url 'ddcz:phorum-list' %}">
+            {% csrf_token %}
+            {{ phorumCommentForm }}
+            <input type="submit" name="addPhorumComment" value="Přidej" class="comment__submit">
+        </form>
+
+    {% else %}
+        <p class="phorum-anonymous-text">
+            Omlouváme se, ale anonymní přispívání bylo zrušeno kvůli některým bezohledným spammerům. Děkujeme za pochopení.
+        </p>
+    {% endif %}
 
     {% pagination discussions %}
-    <div class="article-list">
+    <section class="article-list">        
     {% for comment in discussions %}
-        <div class="comment">
-            <div class="user_icon"></div>
-            <div class="comment_main">
+        <section class="comment">
+            <picture class="user_icon"></picture>
```

Comment:
I think it should stay `<div>` with `<img>` within—[picture is for selecting between multiple sources](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture), which I don't think will be the case here .